acceptance-tests/src/test/resources/clique/clique-prague.json.tpl

```diff
@@ -0,0 +1,155 @@
+{
+  "config": {
+    "chainId": 1337,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "terminalTotalDifficulty":0,
+    "cancunTime":0,
+    "pragueTime":0,
+    "blobSchedule": {
+      "cancun": {
+        "target": 3,
```

Comment:
I think we can and should set the target and max to 0. And I think it should be the same in the real Linea's genesis file later on.

@fab-10 That should also prevent the inclusion, doesn't it?

Reply:
done